docs/features/opencdc-record.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 'OpenCDC record'
```

Comment:
I'm missing a word or two about the constants that we provide for some metadata fields, some are even populated by conduit or the connector SDK. The constants are defined [here](https://github.com/ConduitIO/conduit-commons/blob/a6adcbfe1168a50f5f42fdeafa0353eae789dd8e/proto/opencdc/v1/opencdc.proto#L10-L24) (general opencdc constants) and [here](https://github.com/ConduitIO/conduit-connector-protocol/blob/669c83d26b3f9bc58672e69e29a5d94a53af0e2e/proto/connector/v1/connector.proto#L18-L21) (related to conduit connectors). This is for example how we'll add support for multiple tables, to add another constant here that will be used by convention to define where a record was read from or where it should be sent.